scripts/stac/tests/collection_test.py

```diff
@@ -24,24 +24,24 @@ def setup() -> Generator[CollectionMetadata, None, None]:
         "start_datetime": datetime(2022, 2, 2),
         "end_datetime": datetime(2022, 2, 2),
         "lifecycle": "completed",
-        "event_name": "Forest assessment",
+        "event_name": "Forest Assessment",
         "historic_survey_number": None,
-        "geographic_description": "Auckland North",
+        "geographic_description": "Auckland North Forest Assessment",
```

Comment:
Does the user need to duplicate this information from `event_name` to `geographic_description`?

Reply:
Sorry that was confusing I think I did a better job explaining it here: https://github.com/linz/argo-tasks/pull/845